src/Simulation.jl

```diff
@@ -230,17 +232,43 @@ this outputs a sparse matrix of  num reactions xnum species containing the produ
 rate of that species associated with that reaction
 """
 function rops(bsol::Q,t::X) where {Q<:Simulation,X<:Real}
-    ropmat = spzeros(length(bsol.domain.phase.reactions),length(bsol.domain.phase.species))
```

Comment:
Check that it's not necessary to update other simulation post-processing.

Reply:
Done